docs/services/slam/cartographer/_index.md

```diff
@@ -18,23 +18,23 @@ Install the binary required to utilize the `cartographer` library on your machin
 {{% tab name="Linux aarch64" %}}
 
 ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-sudo curl -o /usr/local/bin/carto_grpc_server http://packages.viam.com/apps/slam-servers/carto_grpc_server-stable-aarch64.AppImage
-sudo chmod a+rx /usr/local/bin/carto_grpc_server
+sudo curl -o /usr/local/bin/cartographer-module http://packages.viam.com/apps/slam-servers/cartographer-module-stable-aarch64.AppImage
+sudo chmod a+rx /usr/local/bin/cartographer-module
 ```
 
 {{% /tab %}}
 {{% tab name="Linux x86_64" %}}
 
 ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-sudo curl -o /usr/local/bin/carto_grpc_server http://packages.viam.com/apps/slam-servers/carto_grpc_server-stable-x86_64.AppImage
-sudo chmod a+rx /usr/local/bin/carto_grpc_server
+sudo curl -o /usr/local/bin/cartographer-module http://packages.viam.com/apps/slam-servers/cartographer-module-stable-x86_64.AppImage
+sudo chmod a+rx /usr/local/bin/cartographer-module
```

Comment:
Just confirming, the user does not need to do this chmod for carto_grpc_server, since the user doesn't directly execute carto_grpc_server?

Reply:
Its b/c the carto_grpc_server executable is embedded within the `cartographer-module` appimage binary.